docs/docs/operator/runtime-threat-detection.md

```diff
@@ -0,0 +1,82 @@
+# Runtime Threat Detection & Response
+The runtime threat detection & response feature of the Kubescape operator enables users to have visibility into the runtime environment and respond to threats in real-time.
+Using eBPF probes toghether with Kubernetes context, the node agent effectively monitors the runtime environment for any suspicious activity.
```

Comment:
```suggestion
Using eBPF probes toghether with Kubernetes context, kubescape effectively monitors the runtime environment for any suspicious activity.
```

Reply:
I'm with David, just would like to see a cpital K on Kubescape.